ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -1536,7 +1536,12 @@ class JITModBuilder {
           }
           assert(CallStack.curMappableFrame());
           tryStopOutlining();
-        } else if (UR->StackAdjust > 0) {
+        } else {
+          // If the stack size hasn't changed, either we've got something wrong,
+          // or the unmappable code has jumped back to mappable code. We could
+          // support the latter, but currently don't.
```

Comment:
s/tt/t/

Reply:
I believe "latter" is correctly spelt.

Reply:
Ah, I thought you were saying "we could support this later", sorry.

"jumped back" isn't quite right though. it could be a forward of backward jump or simply a fall-thru. It's not necessarily "going back" to mappable code that we've seen before, it may be new code.

Reply:
s/back to/into/ ?

Reply:
I'd just say "entered", since there may be no jump involved.

e.g.:

```
foreign_bb:
  nop

mappable_bb:
  nop
```

Reply:
No, "entered" doesn't sound right either.

Reply:
I think "transitioned" is clearer than "entered" because the latter implies call/branch?

Reply:
Yes, I like that.

So something like:

>  If the stack size hasn't changed it means we've transitioned to mappable code via some non-standard control flow (e.g. a jump or a fallthru). We could support this later, but currently don't.

Reply:
(where "could" is optimistic)

Reply:
"Non-standard" seems incorrectly judgmental!

Reply:
See if 8a0ddba works for you.

Reply:
This is fine.

(By non-standard, I meant what you call "BCF": bonkers control flow :) )